app/templating/summary/question.py

```diff
@@ -65,13 +80,15 @@ def _build_answer(cls, question_schema, answer_schema, answers, answers_iterator
     @classmethod
     def _build_checkbox_answers(cls, answer, answer_schema):
         multiple_answers = []
+        CheckboxSummaryAnswer = collections.namedtuple('CheckboxSummaryAnswer', 'label should_display_other')
         for option in answer_schema['options']:
-            if option['value'] in answer:
-                if option['value'] == 'other':
+            if option['label'] in answer:
+                if option['value'].lower() == 'other':
                     summary_option_display_value = cls._get_checkbox_other_display_value(answer, answer_schema, option)
```

Comment:
I think this has got lost in the previous comments. If we don't need the `_get_checkbox_other_display_value` method we should remove it and just use the label.